compiler/hash-typecheck/src/options/normalisation.rs

```diff
@@ -152,7 +183,7 @@ impl NormalisationState {
 pub enum NormalisationMode {
     /// Normalise the term as much as possible.
     Full,
-    /// Normalise the term to a single step.
+    /// Normalise the term to a single step (default).
```

Comment:
perhaps, this can be self documents with just:
```suggestion
    /// Normalise the term to a single step
    #[default]
```